django_project/settings.py

```diff
@@ -17,7 +17,7 @@
 BASE_DIR = Path(__file__).resolve().parent.parent
 
 env = Env(DEBUG=(bool, False))
-env.read_env(env.str('./', '.env'))
+env.read_env(env.str("../antiplag/", ".env"))
```

Comment:
I'm not sure if this is ok. The .env is in current folder (`.`). What if someone renames the root directory? It's not correct to just go one dir up and then back into it.